pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -326,6 +327,9 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 			allVolumesAdded = false
 			continue
 		}
+		if present, err := dswp.desiredStateOfWorld.IsVolumeSeenByDSW(uniquePodName, volumeSpec); err == nil && present {
+			volumeAdded = true
```

Comment:
by calling IsVolumeSeenByDSW, it will call getUniqueVolumeName(). Then during dswp.desiredStateOfWorld.AddPodToVolume() call, it will call getUniqueVolumeName(). Maybe you could improve this by only calling it once?

Reply:
IsVolumeSeenByDSW can return the UniqueVolumeName so that the unique name is passed to AddPodToVolume().

@jingxu97 
Since there are many tests where AddPodToVolume() is called, can you confirm this approach before I make the change ?